### PR TITLE
Fixing overlap in the wallet details header and wallets list item name

### PR DESCRIPTION
**Pull request by banklesss**

### CODE REVIEW

File: src/navigation.ts [hunk 1]
```diff
@@ -28,8 +28,8 @@ export const defaultStackNavigationOptionsV2: StackNavigationOptions = {
     fontSize: 16,
     fontFamily: 'Rubik-Medium',
   },
-  headerRightContainerStyle: {
-    paddingRight: 12,
+  headerTitleContainerStyle: {
+    marginHorizontal: 45,
```
Comment (stackchain) [inline src/navigation.ts]: We avoid using margin, margin is evil.

**Review by stackchain — CHANGES_REQUESTED** [review]

File: src/navigation.ts (hunk 1 above)
Comment (banklesss) [reply src/navigation.ts]: That header title is absolute positioned by default, thats the reason why was overlapped. I'm thinking of adding a width of 70% to the full width of the screen. what do you think?

Comment (wolverineks) [reply src/navigation.ts]: percentages are hacks
margins are terrible hacks
react navigation is hacky
hack if you must

**Review by wolverineks — APPROVED** [review]

File: src/navigation.ts (hunk 1 above)
Comment (banklesss) [reply src/navigation.ts]: hahaha. Ok I will add a 70% of width if everyone agrees

**Pull request merged**